### ISSUE TITLE
Add Support for font-feature-settings

### ISSUE THREAD

**Issue by HughxDev** [LeaVerou/prefixfree#105]
`font-feature-settings: 'liga', 'dlig';` should become `-webkit-font-feature-settings: 'liga', 'dlig';`, etc.

**Comment by LeaVerou** [comment]
Did you test it and it didn't work?

**Comment by HughxDev** [comment]
I did! Are you getting different results?

**Comment by LeaVerou** [comment]
Indeed, it doesn't seem to work, even though it's prefixed. Interesting. Will need to investigate more.

**Comment by HughxDev** [comment]
It doesn’t even prefix for me. I just get this:

![Chrome Dev Tools](http://i.imgur.com/qUApx.png)

…even though I can confirm that Prefixfree loaded successfully.

**Comment by LeaVerou** [comment]
Yes, that's what I mean. Even though the browser requires a prefix, it doesn't get added.

**Comment by LeaVerou** [comment]
Does it only reproduce in WebKit?

**Comment by HughxDev** [comment]
Ah, yeah. Works in Firefox.
